src/main/java/ca/uhn/fhir/jpa/starter/interceptor/LoggerInterceptor.java

```diff
@@ -15,32 +12,54 @@
 import ca.uhn.fhir.rest.api.server.RequestDetails;
 import ca.uhn.fhir.rest.api.server.ResponseDetails;
 import ca.uhn.fhir.rest.server.servlet.ServletRequestDetails;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
 
 @Interceptor
-public class CorrelationIdInterceptor {
+public class LoggerInterceptor {
   private static final String X_CORRELATION_ID = "X-Correlation-Id";
+	private static final String CORRELATION_ID = "correlationId";
```

Comment:
There's a mix of tabs added in the new code and the existing code was using spaces.

Reply:
we should have a pre-commit hook that catches lint/formatting errors

Reply:
Yash, theses have still not be addressed.

Reply:
I'll address this issue in another PR.

Reply:
https://meliorix.atlassian.net/browse/RT-1996?atlOrigin=eyJpIjoiZGYzN2M2OTBhMTExNGU5MDk2MTU5MzY2NzE1M2MzZTMiLCJwIjoiaiJ9

Reply:
No, adding a pre-commit hook is a feature that would be nice to have. The formatting (i.e. mix of tabs and spaces) needs to be fixed in this pull request.